src/route.rs

```diff
@@ -126,6 +126,53 @@ impl Route {
         self
     }
 
+    /// Set the service to be called when this route is hit.
+    ///
+    /// ```
+    /// # use actix_web::{*, dev::*, http::header};
+    /// # use std::{task::{Context, Poll}, pin::Pin, future::Future};
+    /// use futures_util::future::{ok, LocalBoxFuture};
+    ///
+    /// struct HelloWorld;
+    ///
+    /// impl Service<ServiceRequest> for HelloWorld {
+    ///     type Response = ServiceResponse;
+    ///     type Error = actix_web::Error;
+    ///     type Future = LocalBoxFuture<'static, Result<Self::Response, Self::Error>>;
+    ///
+    ///     fn poll_ready(&self, _: &mut Context<'_>) -> Poll<Result<(), Self::Error>> {
+    ///         Poll::Ready(Ok(()))
+    ///     }
```

Comment:
prefer a re-export of always_ready! macro and use here

Reply:
it was already being re-exported :)